tensorrt_llm/_torch/pyexecutor/py_executor_creator.py

```
@@ -222,6 +182,25 @@ def update_sampler_max_seq_len(max_seq_len, sampler):
+        assert tokenizer is not None
+        guided_decoding_config = GuidedDecodingConfig(
+            backend=GuidedDecodingConfig.GuidedDecodingBackend.XGRAMMAR,
+            **_xgrammar_tokenizer_info(tokenizer))
+    elif guided_decoding_backend == 'llguidance':
+        assert tokenizer is not None
+        guided_decoding_config = GuidedDecodingConfig(
+            backend=GuidedDecodingConfig.GuidedDecodingBackend.LLGUIDANCE,
+            **_llguidance_tokenizer_info(tokenizer))
+    elif guided_decoding_backend is not None:
+        raise ValueError(
+            f"Unsupported guided decoding backend {guided_decoding_backend}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)